src/source/PeerConnection/PeerConnection.c

```diff
@@ -992,6 +994,7 @@ STATUS freePeerConnection(PRtcPeerConnection* ppPeerConnection)
     PDoubleListNode pCurNode = NULL;
     UINT64 item = 0;
     UINT64 startTime;
+    UINT32 count;
```

Comment:
(nit) Can we make count variable name more descriptive

Reply:
initialize